atom/app/atom_content_client.h

```diff
@@ -31,6 +31,9 @@ class AtomContentClient : public brightray::ContentClient {
       std::vector<content::PepperPluginInfo>* plugins) override;
   void AddServiceWorkerSchemes(
       std::set<std::string>* service_worker_schemes) override;
+  void AddSecureSchemesAndOrigins(
```

Comment:
I believe it's called by chromium or brightray code. I confirmed by removing it and running tests, which then failed.